scripted-tests/run/java-io-file-1/Platform.scala

```diff
@@ -1,5 +1,5 @@
 // This file is used only when compiling sources using Scala Native
 
 object Platform {
-  val isWindows = scalanative.runtime.Platform.isWindows
+  val isWindows = scalanative.runtime.Platform.isWindows()
```

Comment:
Shouldn't `isWindows` be defined as parenthesis-less in `Platform` too since it not a side-effectful `def` ? Also this could be a good time to do it since we are in the 0.5 breaking window